### ISSUE TITLE
featured chat mode 

### ISSUE THREAD

**Issue by conroygreen** [steveseguin/social_stream#151]
featured-chat index.html doesn't show anything in overlay when I select a message. i am not sure if there something special i need to do.

**Comment by steveseguin** [comment]
Nothing special.  just ensure you open it via the extension, as it comes appended with ?session=XXXXX , and that value needs to match your dock, etc.

If your dock is working, the index should be working.

If you put things into "server" mode, it might not work.  There's some other toggles that might break things also, so perhaps reset the settings or undo changes until it works.

**Comment by conroygreen** [comment]
when I click the link for featured-chat it opens in the browser with just a white background and nothing appears on screen

**Comment by steveseguin** [comment]
it will be white, until you manually select a message in the dock.html page

**Comment by conroygreen** [comment]
this is the dock link "https://socialstream.ninja/dock.html?session=A3uryLXy49&helpermode&chatmode"
and this is the featured-chat link "https://socialstream.ninja/index.html?session=A3uryLXy49"
should the dock be in helper mode?

**Comment by steveseguin** [comment]
I'd probably try to not put it into any special modes; those might disable the ability to feature messages

ie: some modes will only let them "pin" messages, and not "highlight" messages

**Comment by conroygreen** [comment]
yes it works now thanks

**Comment by conroygreen** [comment]
one more question, can whatsApp chat pop up in featured-chat?

**Comment by steveseguin** [comment]
Yes. You need to manually opt in to allow whatsapp to be captured tho.
